app/views/result_filters/qualification/new.html.erb

```diff
@@ -0,0 +1,107 @@
+                </p>
+                <p class="govuk-body">
+                  To teach in the rest of the UK or internationally, you may wish to consider a ‘PGCE (or PGDE) with QTS’ qualification instead.
+                </p>
+              <% end %>
+            </div>
+
+            <div class="govuk-checkboxes__item">
+              <%=
+                form.check_box(
+                  :qualifications,
+                  { 
+                    class: "govuk-checkboxes__input", 
+                    multiple: true, 
+                    data: { qa: 'pgde_pgce_with_qts'}, 
+                    checked: @view.pgde_pgce_with_qts_checked?,
+                  },
+                  "PgdePgceWithQts", 
+                  false
+                )
+              %>
+              <%= form.label :qualifications, value: "PgdePgceWithQts", class: "govuk-label govuk-checkboxes__label" do %>
+                <span class="govuk-!-font-weight-bold govuk-!-margin-bottom-2 govuk-!-display-block">PGCE (or PGDE) with QTS</span>
+                <p class="govuk-body">
+                  A PGCE (postgraduate certificate in education) with QTS allows you to teach in England, Scotland, Wales and Northern Ireland. It’s also recognised internationally. Many PGCE courses include credits that count towards a Master’s degree.
+                </p>
+                <p class="govuk-body">
+                  Some providers offer a PGDE (postgraduate diploma in education) with QTS, which is equivalent to a PGCE. In some cases these courses offer more credits towards a Master’s degree than PGCE courses.
+                </p>
+              <% end %>
+            </div>
+
+            <div class="govuk-checkboxes__item">
+              <%=
+                form.check_box(
+                  :qualifications,
+                  { 
+                    class: "govuk-checkboxes__input",
+                    multiple: true,
+                    data: { qa: 'other'},
+                    checked: @view.other_checked?
+                  },
+                  "Other", 
+                  false
+                )
+              %>
+              <%= form.label :qualifications, value: "Other", class: "govuk-label govuk-checkboxes__label" do %>
+                <span class="govuk-!-font-weight-bold govuk-!-margin-bottom-2 govuk-!-display-block">Further education (PGCE or PGDE without QTS)</span>
+                <p class="govuk-body">
+                  To teach further education you don’t need QTS. Instead you can study for a PGCE or PGDE without QTS.
+                </p>
+              <% end %>
+            </div>
+          </div>
+        </div>
+      </fieldset>
+
+      <div class="govuk-form-group">
+        <%= form.submit "Find courses", name: nil, class: "govuk-button", data: { qa: 'find_courses' } %>
```

Comment:
```suggestion
        <%= form.submit "Find courses", class: "govuk-button", data: { qa: 'find_courses' } %>
```

do we actually need `name: nil`?

Reply:
It avoids 'commit: true' being passed as a parameter it think.